src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message in the `doWork` method should be more informative and indicate that it is an `Excavator` that is working. Consider specifying the action that the Excavator is performing.